packaging/CMakeLists.txt

```diff
@@ -203,28 +218,38 @@ if (WITH_DOCS)
 endif ()
 
 ##
-# Actual packaging
+# General packaging variables.
 ##
 
 set(CPACK_PACKAGE_NAME Halide)
 set(CPACK_PACKAGE_VENDOR Halide)
-set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Halide compiler and libraries")
 set(CPACK_PACKAGE_INSTALL_DIRECTORY ${CPACK_PACKAGE_NAME})
 set(CPACK_PACKAGE_VERSION_MAJOR ${Halide_VERSION_MAJOR})
 set(CPACK_PACKAGE_VERSION_MINOR ${Halide_VERSION_MINOR})
 set(CPACK_PACKAGE_VERSION_PATCH ${Halide_VERSION_PATCH})
 set(CPACK_VERBATIM_VARIABLES YES)
-# set(CPACK_PACKAGE_DESCRIPTION_FILE "${CMAKE_CURRENT_LIST_DIR}/Description.txt")
+set(CPACK_PACKAGE_DESCRIPTION_FILE "${CMAKE_CURRENT_LIST_DIR}/Description.txt")
 # set(CPACK_RESOURCE_FILE_WELCOME "${CMAKE_CURRENT_LIST_DIR}/Welcome.txt")
```

Comment:
nit: I know this was already commented out, but adding a comment about why is always good